### ISSUE TITLE
Problem: old/unmerged branches contain Python 3 work

### ISSUE THREAD

**Issue by sallain** [archivematica/Issues#818]
Review the following branches and decide what to do about them.

* [dev/issue-6079-python3](https://github.com/artefactual/archivematica/tree/dev/issue-6079-python3)
* [dev/issue-py23-compatibility](https://github.com/artefactual/archivematica/tree/dev/issue-py23-compatibility)

---
**For Artefactual use:**
Please make sure these steps are taken before moving this issue from Review to Done:

- All PRs related to this issue are properly linked 👍
- All PRs related to this issue have been merged 👍
- Test plan for this issue has been implemented and passed 👍
- Documentation regarding this issue has been written and it has been added to the release notes, if needed 👍

**Comment by sevein** [comment]
For the record, these stalled pull requests with Py3 work are being closed:
* https://github.com/artefactual/archivematica/pull/1466
* https://github.com/artefactual/archivematica/pull/1469
* https://github.com/artefactual/archivematica/pull/1477
* https://github.com/artefactual/archivematica/pull/1476
* https://github.com/artefactual/archivematica/pull/1532
* https://github.com/artefactual/archivematica-storage-service/pull/480

New tree lives in [https://github.com/artefactual/archivematica/tree/dev/python3](dev/python3) - by @replaceafill.

**Comment by replaceafill** [comment]
I just reviewed the branches referenced in the description. Some of them have already been implemented in the code base or in the `dev/python3` branch. Some have been done in parts of the code that have been removed now or are too Python 3 specific like the shebang updates (we want to maintain Python 2/3 compatibility). And there's one change we're tracking in https://github.com/archivematica/Issues/issues/1280 but we're not planning on tackling on this release. I think it's fine to keep the branches for reference.
